sdk/python/feast/utils.py

```diff
@@ -212,6 +213,28 @@ def _run_pyarrow_field_mapping(
     return table
 
 
+def _get_fields_with_aliases(
```

Comment:
probably would be good to add a test for this but i won't block on it.

Reply:
Don't have a direct test but pull_latest_query tests will use it. I'll add explicit tests in next PRs.